### PR TITLE
Spotify api

### PR DESCRIPTION
**Pull request by avwilliams1995**
server able to fetch song with name or url, artist, playlist and album data from spotify api and respond back to frontend

### CODE REVIEW

File: apiFetch/apiFetch.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(body),
+  })
+    .then((r) => r.json())
+    .then((response) => {
+      id = response;
+    })
+    .catch((error) => {
+      console.log("error in UserData fetch", error.message);
```
Comment (cyburns) [inline apiFetch/apiFetch.tsx:28]: Overall we are going to need before error handling than console logs. What if a user is on the production version off the app and is trying to get some Spotify info and its not working. They should definitely see an error screen or at least and Alert to try again. https://reactnative.dev/docs/alert

File: package.json [hunk 2]
```diff
@@ -4,26 +4,33 @@
   "version": "1.0.0",
   "scripts": {
     "start": "expo start",
-    "android": "expo start --android",
-    "ios": "expo start --ios",
+    "android": "expo run:android",
+    "ios": "expo run:ios",
```
Comment (cyburns) [inline package.json:8]: why did you change this scripts?

Comment (avwilliams1995) [reply package.json:8]: We can change it back. Was testing something with using the app on my phone I saw online.

**Review by cyburns — APPROVED** [review]
Looks good
